lib/vscode/coder.js

```diff
@@ -1,66 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Coder Technologies. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
 // This must be ran from VS Code's root.
-const gulp = require("gulp");
-const path = require("path");
-const _ = require("underscore");
-const buildfile = require("./src/buildfile");
-const common = require("./build/lib/optimize");
-const util = require("./build/lib/util");
+const gulp = require('gulp');
```

Comment:
❓ Did they change to ' from " upstream?

Reply:
This is a case where our prettier formatting didn’t align with upstream’s behavior